src/js/providers/html5.js

```diff
@@ -502,14 +526,19 @@ define([
                 });
             }
 
-            if (_canSeek) {
+            if (_getSeekableEnd() >= seekPos) {
+                // We can seek if the end seekable range is at or past the seek position
+                _loadInProgress = true;
+            }
+
+            if (_loadInProgress) {
                 _delayedSeek = 0;
-                // handle readystate issue
-                var status = utils.tryCatch(function() {
+                // setting currentTime can throw an exception if video.readyState != 4
```

Comment:
`// setting currentTime can throw an invalid DOM state exception if the video is not ready`